src/Nowruz/modules/userProfile/components/mainInfo/index.tsx

```diff
@@ -72,7 +72,7 @@ export const MainInfo = () => {
       {type === 'users' && user.languages && <LanguageJSX items={user.languages || []} />}
       {org.industry && renderData('Industry', 'globe-04', org.industry)}
       {size && renderData('Size', 'users-01', size)}
-      {org.website && <Website website={org.website ?? ''} />}
+      {org.website && <Website url={org.website ?? ''} />}
```

Comment:
@bastantebom this is what I meant when I'm telling don't touch old components you just fix this component on Org profile and it's been used here also and Thanks to @marjan-ara it's been fixed

Reply:
@jeyem I told you already this is not an old component, this is new! I created this component. I know the usage of this

Reply:
We already discussed this on other PR. @jeyem

Reply:
<img width="884" alt="Screenshot 2024-01-21 at 12 56 33 pm" src="https://github.com/socious-io/web-app-v2/assets/5590282/c847300d-51c0-4856-b7f2-83ed358bd381">